okhttp/src/main/java/okhttp3/internal/http/HttpDate.kt

```diff
@@ -65,55 +63,51 @@
       "EEE, dd-MM-yyyy HH:mm:ss z",
 
       /* RI bug 6641315 claims a cookie of this format was once served by www.yahoo.com */
-      "EEE MMM d yyyy HH:mm:ss z",
-  };
+      "EEE MMM d yyyy HH:mm:ss z")
```

Comment:
According to [kotlin official guide](https://kotlinlang.org/docs/reference/coding-conventions.html#method-call-formatting), you should write like:

```kotlin
      "EEE MMM d yyyy HH:mm:ss z"
    )
```